src/settings/SettingsScreen.js

```diff
@@ -51,13 +53,31 @@ export default function SettingsScreen(props: Props): Node {
           dispatch(setGlobalSettings({ browser: value ? 'embedded' : 'external' }));
         }}
       />
-      <SwitchRow
-        label="Do not mark messages read on scroll"
-        value={doNotMarkMessagesAsRead}
-        onValueChange={value => {
-          dispatch(setGlobalSettings({ doNotMarkMessagesAsRead: value }));
-        }}
-      />
+      <View
+        style={[
+          styles.listItem,
+          {
+            height: 56,
```

Comment:
Why 56?

Reply:
sorry about that, I should have mentioned it earlier. I just made a rough UI for testing the feature. I was planning to make a proper code for that after I made the correct implementation.